editor/editor_node.cpp

```diff
@@ -6828,6 +6833,9 @@ EditorNode::EditorNode() {
 		// Add the driver to the UI.
 		rendering_driver->add_item(driver);
```

Comment:
It's probably should use `EditorPropertyNameProcessor::process_name` (and have driver names added to the capitalization table) to convert it to `OpenGL3`, `D3D12` and `Vulkan`.

Reply:
Makes sense, but that code hasn't been added in this PR. I believe this should be addressed in a separate PR.